docs/api-reference/core/view.md

```diff
@@ -81,18 +81,37 @@ The `viewState` property is intended to support a number of use cases:
 * Overriding a partial set of view state properties from a selected view state.
 
 
-#### `clear` (boolean | object, optional) {#clear}
+#### `clear` (boolean, optional) {#clear}
 
-Clears the contents (pixels) of the viewport. The value of the `clear` prop is passed as an argument to luma.gl's `clear` function. If `true` clears color and depth buffers. If an object, behaviour is controlled by the following fields:
-
-* `color` (boolean or Array) - if not `false`, clears all active color buffers with either the provided color or the currently set clear color.
-* `depth` (boolean)  - if `true`, clears the depth buffer.
-* `stencil` (boolean) - if `true` clears the stencil buffer.
+Clears the contents (pixels) of the viewport. If `true` clears color and depth buffers.
 
 Note that deck.gl always clears the screen before each render, and clearing, while cheap, is not totally free. This means that viewports should only specify the `clear` property if they need additional clearing, e.g. because they are rendering on top of another viewport, or want to have a different background color etc.
 
-Default `false`.
+**Interaction between clearing properties:**
+* `clear: false` - Disables all clearing operations for this view, regardless of `clearColor` or `clearDepth` settings.
+* `clear: true` - Clears both color and depth buffers.
+* `clearColor` - When defined, specifies the color used for clearing, but only has an effect if `clear` is not `false` (i.e., `true` or `undefined`).
+* `clearDepth` - When defined, controls depth buffer clearing, but only has an effect if `clear` is not `false` (i.e., `true` or `undefined`).
```

Comment:
I want to remark that these prop interactions are a bit complicated.

I wonder if it would be simpler if `clearColor` and `clearDepth` settings overrode `clear: false` instead of the other way around (the current version)?

Basically, should `clearColor` setting take priority, or `clear: false`?

`clear: true` is more obvious -- the buffer properties should override.

Zooming out, the `clear` property mainly remains for backwards compatibility and convenience so that users don't need to set `clearColor: [0,0,0,0]` and `clearDepth: true`

Reply:
Circling back to the interleaved case,

The default is not to clear, so the [default views](https://github.com/visgl/deck.gl/blob/e25d735d2d6d8b611a65e60b3b934ab407d006cd/modules/mapbox/src/deck-utils.ts#L201-L206) made for interleaved rendering should continue to work as before.

I need to check what appending a overlapping view with clearing enabled does. I wonder if deck ever supported the combination of a minimap with clearing, maplibre and an interleaved deck view?